cmd/podman/shared/funcs.go

```diff
@@ -8,15 +8,19 @@ import (
 
 	"github.com/containers/libpod/pkg/util"
 	"github.com/google/shlex"
+	"github.com/sirupsen/logrus"
 )
 
 func GetAuthFile(authfile string) string {
-	if authfile != "" {
-		return authfile
+	if authfile == "" {
+		authfile = os.Getenv("REGISTRY_AUTH_FILE")
 	}
 
-	authfile = os.Getenv("REGISTRY_AUTH_FILE")
 	if authfile != "" {
+		if _, err := os.Stat(authfile); err != nil {
```

Comment:
I don't think we need the first if `if authfile != "" {` anymore as we check the inverse up top.

Reply:
I need if authfile != ""  , os.GetEnv("REGISTRY_FILE_PATH) can still return ""

Reply:
Oh shoot, yep, you're right.  Never mind!